### PR TITLE
OADP-1015 Two volume backup-restore E2E-test

### PR DESCRIPTION
**Pull request by hhpatel14**

### CODE REVIEW

File: tests/e2e/lib/apps.go [hunk 1]
```diff
@@ -509,6 +512,62 @@ func VerifyBackupRestoreData(artifact_dir string, namespace string, routeName st
 		}
 	}
 
+	if twoVol {
+		volumeFile := artifact_dir + "/volume-data.txt"
+		return varifyVolume(volumeFile, volumeApi, prebackupState, backupRestoretype)
```
Comment (weshayutin) [inline tests/e2e/lib/apps.go:517]: spelling varifyVolume? is the intention "verify"?

File: tests/e2e/lib/apps.go [hunk 2]
```diff
@@ -509,6 +512,62 @@ func VerifyBackupRestoreData(artifact_dir string, namespace string, routeName st
...
+	}
+
+	return nil
+}
+
+//VerifyVolumeData for application with two volumes
+func varifyVolume(volumeFile string, volumeApi string, prebackupState bool, backupRestoretype BackupRestoreType) error {
```
Comment (weshayutin) [inline tests/e2e/lib/apps.go:524]: same.. spelling

File: tests/e2e/lib/apps.go [hunk 3]
```diff
@@ -509,6 +512,62 @@ func VerifyBackupRestoreData(artifact_dir string, namespace string, routeName st
...
+		if _, err := os.Stat(volumeFile); err == nil {
+			volumeBackupData, err := os.ReadFile(volumeFile)
+			if err != nil {
+				return err
+			}
+			os.Remove(volumeFile)
+			fmt.Printf("Data came from volume-file\n %s\n", volumeBackupData)
+			fmt.Printf("Volume Data after restore\n %s\n", volData)
+			dataIsEqual := false
+			for i := 0; i < 5 && !dataIsEqual; i++ {
```
Comment (weshayutin) [inline tests/e2e/lib/apps.go:546]: checking the first five lines.. perfect

**Review by weshayutin — APPROVED** [review]
Looks good! thank you

**Review by kaovilai — APPROVED** [review]
If we just need two volumes in backup then LGTM. Otherwise do we need in either deploymentconfig or deployment to have two volumes?

**Pull request merged**